packages/permissions/src/hooks/use-is-authorized/use-is-authorized.ts

```diff
@@ -2,30 +2,55 @@ import { useApplicationContext } from '@commercetools-frontend/application-shell
 import {
   hasSomePermissions,
   hasEveryPermissions,
+  hasEveryActionRight,
 } from '../../utils/has-permissions';
 
 type TPermissionName = string;
 type TPermissions = {
   [key: string]: boolean;
 };
 
+type TActionRight = {
+  [key: string]: boolean;
+};
+type TActionRights = {
+  [key: string]: TActionRight;
+};
+
+type TActionRightName = string;
+type TActionRightGroup = string;
+type TDemandedActionRight = {
+  group: TActionRightGroup;
+  name: TActionRightName;
+};
+
 // Forward-compatibility with React Hooks 🎉
```

Comment:
Nitpick: we can remove this

Reply:
I can do that in a follow up PR for all hooks.